src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -200,17 +205,47 @@ export class GraphStyles {
     const isOutside = node.isOutside;
 
     let icons = '';
+    // Put a little more space between icons when a badge has multiple icons
+    const iconMargin = () => (icons === '' ? style(defaultIconMargin) : style({ marginRight: '2px' }));
```

Comment:
This is a closure that uses the `icons` var from line 207. I can move this up but would just need to pass `icons` to the function instead of using it from the outside scope.

Reply:
Yes, I see that,  totally up to you.  For me it just looked a little out of place but you can leave it as-is if you like.